### DISCUSSION TITLE
How to disable the autoscroll when the mouse hovers the bottom of the virtual select

### DISCUSSION THREAD

**Discussion by asiby** [sa-si-dev/virtual-select#82] [Q&A] [answered]
If your virtual select is scrollable, it often unexpectedly scrolls down every time the mouse moves near or over the bottom edge of the virtual select. For very long lists, this can quickly become a UX issue and cause user frustration.

Is there any way to disable it?

Thanks

**Comment by sa-si-dev** [comment]
@asiby, I will fix this issue in the future release. Thanks for letting me know it.

**Comment by sa-si-dev** [comment, marked as answer]
@asiby this issue has been fixed in the release [v1.0.20](https://github.com/sa-si-dev/virtual-select/releases/tag/v1.0.20)

**Reply by asiby** [reply]
Thank you @sa-si-dev

**Comment by foxy17** [comment]
This is still happening with the latest release @sa-si-dev 
It happens for both directions depending on where the cursor is.
